src/types/mod.rs

```diff
@@ -119,11 +119,11 @@ pub enum Type {
 impl fmt::Display for Type {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match *self {
-            Type::Null => write!(f, "Null"),
-            Type::Integer => write!(f, "Integer"),
-            Type::Real => write!(f, "Real"),
-            Type::Text => write!(f, "Text"),
-            Type::Blob => write!(f, "Blob"),
+            Type::Null => f.pad("Null"),
```

Comment:
This is unrelated but allows us to work in cases where a align/width/etc format args are provided. Admittedly, IDK how relevant that is here, but it's better to use Formatter::pad for this sort of thing and I noticed it.